### DISCUSSION TITLE
Meep integration

### DISCUSSION THREAD

**Discussion by mniehus** [HelgeGehring/gdshelpers#52] [General]
In the most recent update a 'meep integration' is mentioned. Is there any documentation on this feature available or planned? cheers

**Comment by mniehus** [comment]
... in particular, it would be interesting for me to understand on whch level the meep simulation is integrated: on the final 'gdsii object/cell/file level'? or rather on the 'gdshelper object level' ? are there any hints on this in the current gdshelper codebase? thank You very much.

**Comment by HelgeGehring** [comment]
The best documentation at the moment are the examples within the same file blow the definition of the class https://github.com/HelgeGehring/gdshelpers/blob/master/gdshelpers/simulation/simulation.py#L174 I hope that helps.
You can directly put shapes of the parts into the simulation, define the heights/refractive indeces/... and put monitors at ports.
I also plan to explain it in the documentation. Feedback about the meep-integration would be highly appreciated!
